microsoft-edge/index.yml

```
@@ -82,7 +82,7 @@ conceptualContent:
 # Supports up to 3 subsections
   sections:
     - title: Microsoft Edge for Business # < 60 chars (optional)
-      # summary:  # < 160 chars (optional)
+      summary: For IT Administrators.  Enterprise = Business = Deploy. # < 160 chars (optional)
```

Not a fan of the short form with the equal signs. Maybe this was just temporary.
Can we do a long sentence:

```suggestion
      summary: Learn to deploy and configure Microsoft Edge for Business # < 160 chars (optional)
```